php/php.blade/src/org/netbeans/modules/php/blade/editor/BladeBracesMatcher.java

```diff
@@ -104,35 +104,21 @@ public int[] findMatches() throws InterruptedException, BadLocationException {
         }
         String tokenText = originToken.getText();
 
-        switch (currentDirection) {
-            case CURLY_START_TO_END -> {
-                return findCloseTag();
-            }
-            case CURLY_END_TO_START -> {
-                return findOpenTag();
-            }
-            case START_TO_END -> {
-                return findDirectiveEnd(tokenText);
-            }
-            case CUSTOM_START_TO_END -> {
-                return findCustomDirectiveEnd(tokenText);
-            }
-            case END_TO_START -> {
-                return findOriginForDirectiveEnd(tokenText);
-            }
-        }
-        
-        return null;
+        return switch (currentDirection) {
+            case CURLY_START_TO_END -> findCloseTag();
+            case CURLY_END_TO_START -> findOpenTag();
+            case START_TO_END -> findDirectiveEnd(tokenText);
+            case CUSTOM_START_TO_END -> findCustomDirectiveEnd(tokenText);
+            case END_TO_START -> findOriginForDirectiveEnd(tokenText);
+            default -> null;
```

Comment:
Please don't use `default` because `currentDirection` is the enum (Add all cases).
See: https://github.com/apache/netbeans/pull/7618#discussion_r1699881123